Bylaws.md

```diff
@@ -54,32 +54,33 @@ Chartered Working Group (CWG) Members are automatically granted Forum Membership
 
 Categories of Forum Membership are defined as follows:
 
-* Voting Member, which contains the following sub-categories:
-   * Certificate Issuer: The member organization operates a certification authority that meets the requirements for membership as a Certificate Issuer in one or more CWGs.
-   * Certificate Consumer: The member organization meets the requirements for membership as a Certificate Consumer in one or more CWGs.
-* Associate Member: As defined in Section 3.1
-* Interested Party: As defined in Section 3.2
-* Probational Certificate Issuer: A Certificate Issuer applying for membership but awaiting full qualification under Section 2.1. During the probational state, the organization has no voting rights. The Membership of a Probational Certificate Issuer is valid for 12 months and may be renewed as decided by the CWG.
+* Voting Member, which contains the following two (2) sub-categories:
+   * Certificate Issuer: The member organization operates a certification authority and meets the requirements for membership as a Certificate Issuer in one or more CWGs.
+   * Certificate Consumer: The member organization produces software that processes certificates and meets the requirements for membership as a Certificate Consumer in one or more CWGs.
+Other types of membership consists of the following three (3) sub-categories:
+   * Associate Member: As defined in Section 3.1
+   * Interested Party: As defined in Section 3.2
+   * Probationary Member (Certificate Issuer/Certificate Consumer): As defined in Section 3.3
```

Comment:
The highlighted text should be applicable to both types of voting members (at least for the probationary member) and not just Certificate Consumers, but its written (but not listed) as if this is a subcategory of a "voting member"?

I have included a suggestion below.

![image](https://github.com/cabforum/forum/assets/722070/2c4b3000-85c1-48e9-8a96-b2070e827a3f)

```suggestion
* Voting Member, which contains the following two (2) sub-categories:
   * Certificate Issuer: The member organization operates a certification authority and meets the requirements for membership as a Certificate Issuer in one or more CWGs.
   * Certificate Consumer: The member organization produces software that processes certificates and meets the requirements for membership as a Certificate Consumer in one or more CWGs.
* Probationary Member: As defined in Section 3.3
   * Certificate Issuer: The member organization operates a certification authority and does not meet all qualifying criteria for membership as a Certificate Issuer in one or more CWGs.
   * Certificate Consumer: The member organization produces software that processes certificates and does not meet all qualifying criteria for membership as a Certificate Consumer in one or more CWGs.
* Associate Member: As defined in Section 3.1
* Interested Party: As defined in Section 3.2
```